### PR TITLE
Enable AppInsights logging

### PR DESCRIPTION
**Pull request by KarolJastrzebski**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -29,7 +34,11 @@ subprojects {
   }
 
   dependencies {
-    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '1.5.0'
+    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
```
Comment (nitinprabhu) [inline build.gradle]: Don't we need instrumentation key config if we enable app insights?

Comment (nitinprabhu) [reply build.gradle]: Also `applicationinsights-agent-2.0.0-BETA.jar` needs to be in lib ?

Comment (jasonpaige) [reply build.gradle]: as long as it's not testing in master we can test in the demo branch?

File: build.gradle [hunk 2]
```diff
@@ -6,6 +6,11 @@ ext {
   springBootVersion = '1.5.6.RELEASE'
 }
 
+def versions = [
+  reformLogging: '1.6.1',
```
Comment (doncem) [inline build.gradle]: best to go with latest - 2.0.2

File: build.gradle (hunk 1 above)
Comment (doncem) [reply build.gradle]: as long as integration tests pass and app boots up - addition of this library is considered a success 🏆

**Pull request merged**